cmake/modules/SwiftSource.cmake

```diff
@@ -126,6 +144,35 @@ function(add_swift_source_group sources)
     HEADER_FILE_ONLY true)
 endfunction()
 
+# Generate an output file map for the given sources
```

Comment:
For those who don't know... is there a description of what an output file map is? How it is used/etc. Is it used anywhere else? Maybe can you add a small blurb here. Your choice though.